### PR TITLE
Added GitHub pull request template

### PR DESCRIPTION
**Pull request by stuartmccoll**
This pull request adds a pull request template to be used on GitHub.

### CODE REVIEW

**Comment by gidsg** [comment]
Can we add a heading _Link to Trello card (if applicable)_?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+## Further comments
+
+If this is a large or complex change, discuss why the solution was chosen and 
+alternatives that were considered, etc.
+
+## Checklist
+
+- [ ] Build (`bundle exec rails server`) runs locally
+- [ ] Added tests for new functionality, or updated existing tests
+- [ ] All tests passing
```
Comment (gidsg) [inline .github/pull_request_template.md:16]: Don't think this is really necessary as CI will have to run and pass before PR can be merged!

**Comment by gidsg** [comment]
Can we add a heading for _Screenshot (if applicable)_

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+be sure to link to that issue.
+
+## Further comments
+
+If this is a large or complex change, discuss why the solution was chosen and 
+alternatives that were considered, etc.
+
+## Checklist
+
+- [ ] Build (`bundle exec rails server`) runs locally
```
Comment (gidsg) [inline .github/pull_request_template.md:14]: Can we just assume this has happened? You'd have to be quite brave to raise a PR without ever having run the code!

File: .github/pull_request_template.md (hunk 1 above)
Comment (stuartmccoll) [reply .github/pull_request_template.md]: 👍

File: .github/pull_request_template.md (hunk 2 above)
Comment (stuartmccoll) [reply .github/pull_request_template.md]: 👍

**Pull request merged**